rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/decompiler/FernflowerDecompilerOperation.java

```diff
@@ -314,7 +314,7 @@ private void handleDecompiledJavaFile(FileModel decompiledFileModel, FileService
             queueSize.incrementAndGet();
             executorService.submit(saveDecompiledRunnable);
 
-            while (queueSize.get() > 1000)
+            while (queueSize.get() > 100000)
```

Comment:
why 100K ? Is that calculated or from experienced testing?

Reply:
Yes, I was finding that this queue was getting exceptionally large on some of the large batchs. I think this was largely the result of analyzing obfuscated code, but I'm not sure that I see a harm to a large queue size here.